src/Kernel/Context.class.st

```diff
@@ -377,6 +377,11 @@ Context >> basicSize [
 	^self primitiveFail
 ]
 
+{ #category : #testing }
+Context >> belongsToDoIt [
+	^self home method isDoIt
```

Comment:
In previous PR I used "context method isDoIt". But it is always false for contexts of blocks. Here it is a general version. There will be PRs to NewTools to reuse it there.